### PR TITLE
fix(build): explicitly expose api/extractors

### PR DESCRIPTION
**Pull request by thekip**
Fixes https://github.com/lingui/js-lingui/issues/1348

### CODE REVIEW

File: scripts/test-node-api.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const assert = require("assert")
+assert(require.resolve('@lingui/cli/api'));
+assert(require.resolve('@lingui/cli/api/extractors/typescript.js'));
+assert(require.resolve('@lingui/cli/api/extractors/babel.js'));
+
+assert(require.resolve('@lingui/core'));
+assert(require.resolve('@lingui/core/compile'));
```
Comment (thekip) [inline scripts/test-node-api.js:7]: kinda quick and dirty smoke test that at least all entries we consider public are really exposed.

**Review by Martin005 — CHANGES_REQUESTED** [review]

File: packages/cli/package.json [hunk 2]
```diff
@@ -28,6 +28,12 @@
     },
     "./api": {
       "require": "./build/api/index.js"
+    },
+    "./api/extractors/babel.js": {
+      "require": "./build/api/extractors/babel.js"
+    },
+    "./api/extractors/typescript.js": {
```
[the remark this answers is not in the document]
Comment (thekip) [reply packages/cli/package.json:35]: You are right, could you change this together with a test  because I will not able to do it quick.

Comment (Martin005) [inline packages/cli/package.json:35]:
```suggestion
    "./api/extractors/typescript": {
```

File: packages/cli/package.json [hunk 3]
```diff
@@ -28,6 +28,12 @@
     },
     "./api": {
       "require": "./build/api/index.js"
+    },
+    "./api/extractors/babel.js": {
```
Comment (Martin005) [inline packages/cli/package.json:32]:
```suggestion
    "./api/extractors/babel": {
```

File: scripts/test-node-api.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const assert = require("assert")
+assert(require.resolve('@lingui/cli/api'));
+assert(require.resolve('@lingui/cli/api/extractors/typescript.js'));
```
Comment (Martin005) [inline scripts/test-node-api.js:3]:
```suggestion
assert(require.resolve('@lingui/cli/api/extractors/typescript'));
```

File: scripts/test-node-api.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+assert(require.resolve('@lingui/cli/api/extractors/babel.js'));
```
Comment (Martin005) [inline scripts/test-node-api.js:4]:
```suggestion
assert(require.resolve('@lingui/cli/api/extractors/babel'));
```

**Review by Martin005 — APPROVED** [review]

**Pull request merged**